src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegrationInDirectory.tsx

```diff
@@ -200,3 +199,38 @@ const IntegrationFlex = styled('div')`
 const IntegrationItemBox = styled('div')`
   flex: 1;
 `;
+
+const IntegrationStatus = styled((props: {status: ObjectStatus}) => {
+  const {status, ...p} = props;
```

Comment:
@NisanthanNanthakumar If `props` only has the type `{status: ObjectStatus}`, then `...p` is going to be an empty object here. So either the input prop should be modified or using the spread operator here is unnecessary.

Reply:
@scefali the `...p` refers to the styles of the styled component. It is used to apply the styles to the the inner `<div>`.